README.md

```diff
@@ -56,12 +58,56 @@ cd sage
 cargo run --release tests/config.json 
 ```
 
+## Installing the latest release
```

Comment:
Likewise, "Downloading the latest release"

Reply:
done